CodeEdit/Features/CEWorkspace/Views/Pages/TasksCEWorkspaceSettingsView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  TasksCEWorkspaceSettingsView.swift
+//  CodeEdit
+//
+//  Created by Axel Martinez on 27/3/24.
+//
+
+import SwiftUI
+
+struct TasksCEWorkspaceSettingsView: View {
+    let workspace: WorkspaceDocument
+
+    // TODO: Separate Project Settings from Task Settings
+    @Binding var projectSettings: CEWorkspaceSettingsData.ProjectSettings
+    @Binding var settings: CEWorkspaceSettingsData.TasksSettings
+
+    @State private var selectedTaskId: UUID?
+    @State private var addTaskSheetPresented: Bool = false
+
+    var body: some View {
+        SettingsForm {
+            Section {
+                TextField(text: $projectSettings.projectName) {
+                    Text("Name")
+                }
+                Toggle("Tasks", isOn: $settings.enabled)
+            }
+            Section(
+                content: {
+                    if settings.items.isEmpty {
+                        Text("No tasks")
+                            .foregroundColor(.secondary)
+                            .frame(maxWidth: .infinity, alignment: .center)
+                    } else {
+                        ForEach(settings.items) { task in
+                            HStack {
+                                Text(task.name)
+                                Spacer()
+                                Group {
+                                    Text(task.command)
+                                    Image(systemName: "chevron.right")
+                                }
+                                .font(.system(.body, design: .monospaced))
+                            }
+                            .contentShape(Rectangle())
+                            .onTapGesture {
+                                self.selectedTaskId = task.id
+                                self.addTaskSheetPresented.toggle()
+                            }
+                        }
+                    }
+                }, header: {
+                    Text("Tasks")
+                }, footer: {
+                    HStack {
+                        Spacer()
+                        Button("Add Task...") {
```

Comment:
```suggestion
                        Button("Add Task") {
```

Reply:
This is the design we agreed on, I think we should comment on this before making this change

Reply:
What do you mean by "agreed"? If we use `Add Task...`, we should also use `Done...`, which seems a bit odd. I'm not very familiar with the concept of having three dots behind text with buttons; I've only seen it in TextFields.

Reply:
I've added a mention to you on discord so you can join the thread where this feature is being discussed, and there we can agree on modifications on the current design.